### PR TITLE
[KW-Corpus] Add setup details

### PR DESCRIPTION
**Pull request by ifosch**

### CODE REVIEW

**Comment by christofdamian** [comment]
+1

File: keyword-corpus/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+In order to setup the stack, first step is to [quick install Miniconda](http://conda.pydata.org/docs/install/quick.html), which depends on the platform being used. There are install wizards/howto's for Linux, MacOS, and Windows, though the last one might be a little bit more tricky.
+
+Another obvious, but required, step is to clone the [Research repository](https://github.com/Devex/research) and step down to the keyword-corpus directory.
+This can be achieved through the following command:
+
+    git clone https://github.com/Devex/research
+
+Once [Miniconda](http://conda.pydata.org/miniconda.html) has been setup, by running the following command the environment should be setup:
+
+    conda create -f environment.yml
```
Comment (christofdamian) [inline keyword-corpus/README.md]: these seems to need a ```-n``` option

**Comment by psyfear** [comment]
+1

**Comment by dsager** [comment]
refs #29

**Pull request merged**